apps/passport-client/src/appHooks.ts

```diff
@@ -0,0 +1,57 @@
+import { User } from "@pcd/passport-interface";
```

Comment:
this file defines a bunch of commonly-used hooks for subscribing to changes to particular subfields of `AppState`. this is basically a simple re-implementation of redux.